### DISCUSSION TITLE
ipsec statusall report interpretation

### DISCUSSION THREAD

**Discussion by tet-web-dev** [strongswan/strongswan#1893] [General]
I'm not understanding the information that is produced when I run the ipsec statusall command.

```
Listening IP addresses:
  172.31.89.153
  20.0.0.1
Connections:
XYX-IKEv2-VPN:  %any...<HOST_PUBLIC_IP>  IKEv1/2
XYX-IKEv2-VPN:   local:  [OMITTED] uses EAP_MSCHAPV2 authentication
XYX-IKEv2-VPN:   remote: uses public key authentication
XYX-IKEv2-VPN:   child:  dynamic === 172.30.101.11/32 TUNNEL
Security Associations (1 up, 0 connecting):
XYX-IKEv2-VPN[1]: ESTABLISHED 103 seconds ago, 172.31.89.153[OMITTED]...<HOST_PUBLIC_IP>[O=WatchGuard, OU=Fireware, CN=ike2muvpn Server]
XYX-IKEv2-VPN[1]: IKEv2 SPIs: 602e4acf58c75ed1_i* 62b13ef6aa4aba44_r, EAP reauthentication in 2 hours
XYX-IKEv2-VPN[1]: IKE proposal: AES_CBC_256/HMAC_SHA2_256_128/PRF_HMAC_SHA2_256/MODP_2048
XYX-IKEv2-VPN{1}:  INSTALLED, TUNNEL, reqid 1, ESP in UDP SPIs: c9370c1b_i af183154_o
XYX-IKEv2-VPN{1}:  AES_CBC_256/HMAC_SHA2_256_128, 0 bytes_i, 0 bytes_o, rekeying in 46 minutes
XYX-IKEv2-VPN{1}:   10.184.123.129/32 === 172.30.101.11/32 *** <----- this line!
```
**What is that last line telling me?** 172.30.101.11 is the address configured under rightsubnet
falling back to my coding background I understand "===" to mean they are the same thing. I can't ping the 172.30.101.11 address but I CAN ping 10.184.123.129 and it is an alias of the the machine itself (localhost)

**Comment by tobiasbrunner** [comment]
It tells you what traffic selectors were negotiated (or configured if you look under `Connections:`). `===` is just a separator for the two ends.

> I can't ping the 172.30.101.11 address

If it's not an IP address on the peer itself, have a look at [the docs](https://docs.strongswan.org/docs/5.9/howtos/forwarding.html).

**Reply by tet-web-dev** [reply]
Ahh yes I probally have that exact page you linked open in 3 tabs. I think the parts in bold fit my scenario because my right and rightsubnet are different but I'm not following the suggestion.
>  If the VPN gateway is the default gateway of the accessed LAN nothing special has to be done. **If this is not the case, either add a route on all hosts behind the gateway telling them that** the subnet from which virtual IP addresses are assigned to roadwarriors **or remote subnets in site-to-site scenarios can be reached through the VPN gateway.**

As I understand this saying to make routes on my side? (left/client)? looks like it has already been by strongswan on my 220 table:
`172.30.101.11 via 172.31.80.1 dev ens5 proto static src 10.184.123.129 ` but on the same note this is confusing because 172.31.80.1 is my default gateway on the left and if traffic went from my 10.184 to the vpn destination of 172.20.101.11 it would wouldn't be encrypted.

**Reply by tobiasbrunner** [reply]
You might want to read about IPsec policies in [the introduction](https://docs.strongswan.org/docs/5.9/howtos/introduction.html#_ike_and_ipsec_basics). The problem is probably more on the other end (forwarding, firewall, routing...).
